### ISSUE TITLE
Top Pen button support

### ISSUE THREAD

**Issue by gerroon** [flxzt/rnote#1243]
Hi

It would be nice if the Wacom top pen button is supported so that the user can pan the UI up and down also use  it to pan the canvas. It seems like pressing the top pen button doesn't do any action. I can use the bottom one for erasing but the top one doesn't switch to the tool that is set in the prefs.

![image](https://github.com/user-attachments/assets/a785dd8e-c508-4469-b64d-bcd7861591f4)

**Comment by Doublonmousse** [comment]
Can I see what's the configuration panel like ?
Is there a windows ink option in the wacom configuration panel and is it activated ?

**Comment by gerroon** [comment]
![image](https://github.com/user-attachments/assets/adacd7fc-cff5-43d8-8f5c-04bcfcd6ba94)

![image](https://github.com/user-attachments/assets/84fd3a80-139a-49e1-a2ab-8690cdd5dc26)

Beside the tool setup, it would be nice if the top button can be also used to move the setting/side panel up and down, it seems like the only way to move it up and down is a muse wheel and these pens dont have mouse wheels.

**Comment by Doublonmousse** [comment]
Yeah, to scroll in the setting/side panel up and down with a pen,the only option right now is to grab the scrollbar with the pen. This is a general gtk4 behavior, maybe I can ask on their gitlab why that is the case (and open an issue if this could be changed).

Can you put "application defined" for the top button in the wacom settings and check if this registers in rnote ?
